src/data/finsupp/finsupp_lattice.lean

```diff
@@ -0,0 +1,128 @@
+import data.finsupp
```

Comment:
This would need (some form of) copyright banner,
and after the import statements, a module docstring.

Reply:
I think I would rename the file to `lattice.lean`, because it is already in the `finsupp` folder.